js/huobi.js

```diff
@@ -4939,9 +4939,11 @@ module.exports = class huobi extends Exchange {
         let network = this.safeStringUpper (params, 'network'); // this line allows the user to specify either ERC20 or ETH
         network = this.safeStringLower (networks, network, network); // handle ETH>ERC20 alias
         if (network !== undefined) {
-            // possible chains - usdterc20, trc20usdt, hrc20usdt, usdt, algousdt
+            // possible chains - algousdt, bep20usdt, cchainusdt, cubeusdt, hrc20usdt, solusdt, trc20usdt, usdt, usdterc20
             if (network === 'erc20') {
                 request['chain'] = currency['id'] + network;
+            } else if (network === 'omni') {
```

Comment:
@sc0Vu Question, instead of having to handle all of these exceptions and manually re-create the chain id every time, can't we just get it from the currencies information when we have the chain + currencyId available?

Example:
![image](https://user-images.githubusercontent.com/43336371/189637712-eb97a580-c808-48be-9b52-9803135d0f01.png)

Reply:
@sc0Vu any update here?